src/frontend-v2/e2e/src/app.e2e-spec.ts

```diff
@@ -0,0 +1,23 @@
+import { AppPage } from './app.po';
+import { browser, logging } from 'protractor';
+
+describe('workspace-project App', () => {
+  let page: AppPage;
+
+  beforeEach(() => {
+    page = new AppPage();
+  });
+
+  it('should display welcome message', () => {
```

Comment:
the e2e specs can be removed (-: but let's keep the component unit tests

Reply:
Removed the e2e folder